javascript/ecmascript-2015/spread-map-set-math/spread-part-2.md

```diff
@@ -55,9 +50,10 @@ Fill in the gaps such that the array is properly filled in:
 const initial = [1, 2, 3];
 const second = [5, 6, 7];
 const final = [
-?????? ???
-???, 
-???];
+  ?????? ???
```

Comment:
Technically this should be:

```suggestion
  ?????????
```

because the answer is

```
...initial,
```

where the tokens are `...`, `initial`, and `,`.

How does this look in the app though? 😅

@kapnobatai137 can you verify on alpha?

Reply:
Tested it, it works. I've committed the suggestion.